TweetMate/src/main/res/values/strings.xml

```diff
@@ -9,252 +9,295 @@
+    <string name="success_cancel_image">Images canceled.</string>
+    <string name="success_apply">Settings applied.</string>
+    <string name="success_copy_tweet">Tweet copied.</string>
 
     <!-- Fail -->
-    <string name="fail_auth">認証に失敗しました…</string>
-    <string name="fail_load">読み込みに失敗しました…</string>
-    <string name="fail_get_user">ユーザーの取得に失敗しました…</string>
-    <string name="fail_get_list">リストの取得に失敗しました…</string>
-    <string name="fail_no_set_wallpaper">壁紙は設定されていません。</string>
-    <string name="fail_no_search">保存した検索はありません。</string>
-    <string name="fail_no_draft">下書きはありません。</string>
-    <string name="fail_no_text">文章が入力されていません。</string>
-    <string name="fail_no_hashtag">ハッシュタグ履歴はありません。</string>
-    <string name="fail_no_list">リストが存在しません。</string>
-    <string name="fail_no_more_add">これ以上追加できません。</string>
-    <string name="fail_no_more_move">これ以上移動できません。</string>
-    <string name="fail_using_column">起動カラムに設定されています。</string>
-    <string name="fail_using_account">使用中のアカウントです。</string>
-    <string name="fail_using_via">使用中のキーです。</string>
-    <string name="fail_already_add">既に追加されています。</string>
-    <string name="fail_need_permission">アクセス許可が必要です。</string>
+    <string name="fail_auth">Failed to authorization…</string>
+    <string name="fail_load">Failed to load…</string>
+    <string name="fail_get_user">Failed to get user…</string>
+    <string name="fail_get_list">Failed to get list…</string>
+    <string name="fail_no_set_wallpaper">No wallpaper is set.</string>
+    <string name="fail_no_search">No saved searches.</string>
+    <string name="fail_no_draft">No drafts.</string>
+    <string name="fail_no_text">No text entered.</string>
+    <string name="fail_no_hashtag">No hashtag history.</string>
+    <string name="fail_no_list">No list exists.</string>
+    <string name="fail_no_more_add">No more can be added.</string>
+    <string name="fail_no_more_move">No more can be moved.</string>
+    <string name="fail_using_column">Startup column cannot be deleted.</string>
+    <string name="fail_using_account">Account in use cannot be released.</string>
+    <string name="fail_using_via">Key in use cannot be released.</string>
+    <string name="fail_already_add">Already added.</string>
+    <string name="fail_need_permission">Access permission is required.</string>
 
     <!-- Sentence -->
-    <string name="sentence_created_by">%1$sさんが作成</string>
-    <string name="sentence_retweeted_by">%1$sさんがリツイート</string>
-    <string name="sentence_members">%1$d人のメンバー</string>
-    <string name="sentence_trouble_auth">ユーザ名またはメールアドレスでログインをお試し下さい。</string>
-    <string name="sentence_notice_auth">アプリを利用するには、\nTwitterアカウントが必要です。\n\n
-        通知機能、ダイレクトメッセージを使用\nする場合は、公式アプリをご利用下さい。</string>
+    <string name="sentence_created_by">Created by %1$s</string>
+    <string name="sentence_retweeted_by">Retweeted by %1$s</string>
+    <string name="sentence_members">%1$d members</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/61661c239c012b88a18da6118962b9568a3a96d6/TweetMate/src/main/res/values/strings.xml#L292"></span>Formatting %d followed by words ("members"): This should probably be a plural rather than a string</td>
    </tr>
  </tbody>
</table>